__tests__/cmds/openapi/reduce.test.ts

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable no-console */
+import fs from 'fs';
+
+import chalk from 'chalk';
+import prompts from 'prompts';
+
+import OpenAPIReduceCommand from '../../../src/cmds/openapi/reduce';
+
+const reducer = new OpenAPIReduceCommand();
+
+const successfulReduction = specPath => `${specPath} has been reduced to output.json! 🤏`;
+
+const testWorkingDir = process.cwd();
+
+let consoleInfoSpy;
+const getCommandOutput = () => consoleInfoSpy.mock.calls.join('\n\n');
+
+describe('rdme openapi:reduce', () => {
+  beforeEach(() => {
+    jest.mock('fs');
+
+    consoleInfoSpy = jest.spyOn(console, 'info').mockImplementation();
+  });
+
+  afterEach(() => {
+    consoleInfoSpy.mockRestore();
+
+    process.chdir(testWorkingDir);
+
+    jest.clearAllMocks();
+  });
+
+  describe('reducing', () => {
```

Comment:
I'm not testing the meat of the reduction library here, just what the command is supplying it, because the reduction library has pretty decent test coverage over in the [`oas` repo](https://github.com/readmeio/oas/blob/main/__tests__/lib/reducer.test.ts).